### ISSUE TITLE
Converting object detector model to TorchScript

### ISSUE THREAD

**Issue by omeryasar** [pytorch/pytorch#46654]
Hi,
Currently I try to convert this https://github.com/yhenon/pytorch-retinanet implentation of retinanet to torchscript so be able to make inference in c++ environment. But faced with some problems. 

P5_x : torch.Tensor = self.P5_1(C5) , in this line I had an error as : Tried to access nonexistent attribute or method 'P5_1' of type 'Tensor (inferred)'.:

So P5_1 is nn.Conv2d(C5_size, feature_size, kernel_size=1, stride=1, padding=0). 

So what should I do to eliminate this error.
I hope question is clear enough. Thanks in advance

**Comment by gmagogsfm** [comment]
Could you provide more details to repro this issue? I wrote a minimal program that's similar to the Conv2d you referred to, but it is compiled without a problem at HEAD.

```
import torch


class TestModule(torch.nn.Module):
    def __init__(self):
        super(TestModule, self).__init__()
        self.conv = torch.nn.Conv2d(4, 256, kernel_size=1, stride=1, padding=0)

    def forward(self, x):
        return self.conv(x)

torch.jit.script(TestModule())
```

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
